shell/platform/common/client_wrapper/core_implementations.cc

```diff
@@ -164,6 +166,51 @@ void ReplyManager::SendResponseData(const std::vector<uint8_t>* data) {
 
 }  // namespace internal
 
+// ========== method_channel.h ==========
+
+namespace internal {
+
+static constexpr char kControlChannelName[] = "dev.flutter/channel-buffers";
+static constexpr char kResizeMethod[] = "resize";
+static constexpr char kOverflowMethod[] = "overflow";
+
+// Adjusts the number of messages that will get buffered when sending messages
+// to channels that aren't fully set up yet. For example, the engine isn't
+// running yet or the channel's message handler isn't set up on the Dart side
+// yet.
```

Comment:
Here and below: per the style guide, declaration comments belong at the declaration point (which is the header), not the definition point. That's true even for things that not intended for general use, like `internal`-namespaced functions or private class methods.